### ISSUE TITLE
Please upgrade pypi2nix from 1.6.0 to 1.8.0 in NixOs 17.0.3

### ISSUE THREAD

**Issue by stefano-m** [NixOS/nixpkgs#26739]
## Issue description

NixOS 17.03 provides [pypi2nix version 1.6.0](https://github.com/garbas/pypi2nix/releases/tag/v1.6.0) (released in December 2016)

However I can't seem to make it work out of the box because when I run it it fails with

```python
ImportError: No module named 'pkg_resources'
```

The latest version of pypi2nix at the time of this writing is [1.8.0](https://github.com/garbas/pypi2nix/releases/tag/v1.8.0) (released in April 2017) which contains a fix for the aforementioned problem (and is already present in the `unstable` nixpkgs channel).

### Steps to reproduce

1. Install pypi2nix from the default channel
```
$ nix-env -i pypi2nix
```
2. Verify it's version 1.6.0
```
$ nix-env -qaPsc pypi2nix
IPS  nixos.pypi2nix  pypi2nix-1.6.0  = 1.6.0
```
3. Running pypi2nix 1.6.0 fails
```
$ pypi2nix
Traceback (most recent call last):
  File "/nix/store/mdjfm8bnxf5dd2lpg0fbdgaad72qyvq4-pypi2nix-1.6.0/bin/.pypi2nix-wrapped", line 2, in <module>
    import pypi2nix.cli
  File "/nix/store/mdjfm8bnxf5dd2lpg0fbdgaad72qyvq4-pypi2nix-1.6.0/pkgs/pypi2nix/cli.py", line 9, in <module>
    import pypi2nix.stage2
  File "/nix/store/mdjfm8bnxf5dd2lpg0fbdgaad72qyvq4-pypi2nix-1.6.0/pkgs/pypi2nix/stage2.py", line 9, in <module>
    import pkg_resources
ImportError: No module named 'pkg_resources'
```
4. Add the unstable channel
```
$ nix-channel --add https://nixos.org/channels/nixpkgs-unstable
$ nix-channel --update
```
5. Verify the available versions
```
$ nix-env -qaPsc pypi2nix
IPS  nixos.pypi2nix    pypi2nix-1.6.0  = 1.6.0
--S  nixpkgs.pypi2nix  pypi2nix-1.8.0  > 1.6.0
```
6. Upgrade pypi2nix
```
nix-env -i pypi2nix
replacing old ‘pypi2nix-1.6.0’
installing ‘pypi2nix-1.8.0’
```
7. Verify that pipy2nix 1.8.0 does not fail
```
pypi2nix --version
1.8.0
```

## Technical details

* System: NixOS: `17.03.1298.5bafc71b61 (Gorilla)`
* Nix version: `nix-env (Nix) 1.11.9`
* Nixpkgs version: `"17.03.1298.5bafc71b61"`
* Sandboxing enabled: `build-use-sandbox = false`

**Comment by FRidh** [comment]
cc @garbas

**Comment by garbas** [comment]
closed with #26746
